data_hub_api/docmaps/v2/codecs/elife_manuscript.py

```diff
@@ -67,11 +67,12 @@ def get_docmap_elife_manuscript_output(
 
 
 def get_elife_manuscript_volume(
-    manuscript_version: ApiManuscriptVersionInput
+    first_manuscript_version: ApiManuscriptVersionInput
 ) -> Optional[str]:
-    rp_publication_year = manuscript_version['rp_publication_year']
-    if rp_publication_year and rp_publication_year >= ELIFE_FIRST_PUBLICATION_YEAR:
-        return str(rp_publication_year - ELIFE_FIRST_PUBLICATION_YEAR)
+    if first_manuscript_version['rp_publication_timestamp']:
```

Comment:
Are there examples where we don't have a publication date but we would generate the partOf information?

Reply:
Actually you are right. If there is no 'rp_publication_timestamp' we are not even providing manuscript-published step. I will fix this.